src/GoogleFitOnFhir/Services/IGoogleFitDataImporter.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GoogleFitOnFhir.Services
+{
+    public interface IGoogleFitDataImporter
+    {
+        Task ImportFitnessData(string userId, CancellationToken cancellationToken);
```

Comment:
nit: I think we should rename this to just `Import`. GoogleFit actually supports more than just fitness data.